lib/sedna/service/client.py

```diff
@@ -26,6 +26,7 @@
 from websockets.exceptions import ConnectionClosedError, ConnectionClosedOK
 
 from sedna.common.log import LOGGER
+from sedna.common.file_ops import FileOps
 
 
 @retry(stop_max_attempt_number=3,
```

Comment:
this contant may put in `constant.py`

Reply:
This may not be appropriate to use a constant instead of the value